libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxHybridSpeechPlayer.kt

```diff
@@ -0,0 +1,91 @@
+package com.mapbox.navigation.ui.voice.api
+
+import android.content.Context
+import com.mapbox.navigation.ui.base.api.voice.SpeechPlayer
+import com.mapbox.navigation.ui.base.api.voice.SpeechPlayerCallback
+import com.mapbox.navigation.ui.base.model.voice.Announcement
+import com.mapbox.navigation.ui.base.model.voice.SpeechState
+import java.util.Queue
+import java.util.concurrent.ConcurrentLinkedQueue
+
+/**
+ * Hybrid implementation of [SpeechPlayer] combining [MapboxOnboardSpeechPlayer] and
+ * [MapboxOffboardSpeechPlayer] speech players.
+ * @property context Context
+ * @property accessToken String
+ * @property language String
```

Comment:
What's the schema?

Reply:
What do you mean by schema here? 🤔

Reply:
How should the passed `string`  look like, is it a specific iso code?

Reply:
Ohh for the language. Got it. It needs to be a `Locale` language (ISO 639). Will add a comment, good point 👍